### PR TITLE
pump icon updates and naming update to better reflect naming schema from boiler and mixer documentation

### PR DESCRIPTION
**Pull request by joanwa**
* Mixer naming
* solar module: add collectorMax/MinTemp
* dampedtemp -> dampedoutdoortemp
* heatPump -> heatingPump (to not confuse with actual heatpumps)
* tankBottomTemp2 -> tank2BottomTemp (temp of second tank, not second temp of tank)
* pumpWorkMin -> pumpWorkTime (to not confuse with minimum in a few lines above)
* many more
*

### CODE REVIEW

**Comment by joanwa** [comment]
this PR probably changes too many small things at once. Anyways, you could review it, and I could break it into smaller pieces with your comments

**Comment by proddy** [comment]
thanks, I'll take a look...

File: src/devices/solar.h [hunk 1]
```diff
@@ -45,21 +45,22 @@ class Solar : public EMSdevice {
...
 
-    int16_t  collectorTemp_          = EMS_VALUE_SHORT_NOTSET; // TS1: Temperature sensor for collector array 1
-    int16_t  tankBottomTemp_         = EMS_VALUE_SHORT_NOTSET; // TS2: Temperature sensor 1 cylinder, tank bottom (solar thermal system)
-    int16_t  tankBottomTemp2_        = EMS_VALUE_SHORT_NOTSET; // TS5: Temperature sensor 2 cylinder, tank bottom, or swimming pool (solar thermal system)
-    int16_t  heatExchangerTemp_      = EMS_VALUE_SHORT_NOTSET; // TS6: Heat exchanger temperature sensor
-    uint8_t  solarPumpModulation_    = EMS_VALUE_UINT_NOTSET;  // PS1: modulation solar pump
-    uint8_t  cylinderPumpModulation_ = EMS_VALUE_UINT_NOTSET;  // PS5: modulation cylinder pump
-    uint8_t  solarPump_              = EMS_VALUE_BOOL_NOTSET;  // PS1: solar pump active
-    uint8_t  valveStatus_            = EMS_VALUE_BOOL_NOTSET;  // VS2: status 3-way valve for cylinder 2 (solar thermal system) with valve
-    int16_t  setpoint_maxBottomTemp_ = EMS_VALUE_SHORT_NOTSET; // setpoint for maximum collector temp
```
Comment (joanwa) [inline src/devices/solar.h:56]: for this one here, I didn't check what `setpoint_maxBottomTemp_`  really sets, comment says collector temp

File: src/devices/mixer.cpp [hunk 2]
```diff
@@ -312,9 +316,9 @@ void Mixer::process_MMStatusMessage(std::shared_ptr<const Telegram> telegram) {
     // 0x21 is position 2. 0x20 is typically reserved for the WM10 switch module
     // see https://github.com/proddy/EMS-ESP/issues/270 and https://github.com/proddy/EMS-ESP/issues/386#issuecomment-629610918
     hc_ = device_id() - 0x20 + 1;
-    changed_ |= telegram->read_value(flowTemp_, 1);         // is * 10
+    changed_ |= telegram->read_value(flowTempLowLoss_, 0); // T0
```
Comment (MichaelDvP) [inline src/devices/mixer.cpp]: For MM10 this is the setpoint calculated by thermostat and send to the mixer, republished here. 
BTW: What is a low loss header?

Comment (joanwa) [reply src/devices/mixer.cpp]: low loss header in german is `hydraulische weiche`, this comes from the MM100 documentation. For the MM100 it's a sensor directly attached to the module.
If MM10 and MM100 use the same id for different values, we should probably differentiate them with different names

Comment (MichaelDvP) [reply src/devices/mixer.cpp]: Yes, i found in MM100 manual that it is a sensor T0. But this single byte value is not T0, measured values are always 16bit. Also Norberts list says for MM100 this is "Requested Flow temperature". I change it back for all mixers to FlowSetTemp and we have to search where T0 is published. It's not in 2D7.. telegrams. MM10 has only a single sensor named FV.

**Comment by MichaelDvP** [comment]
@proddy Due to this name changes i get a lot of mismatches when changing from esp8266 to esp32-v3. I would like to have it mostly compatible. Should i port this to  v3? (i know it's manual merging). I'll also push the latest changes (WPA2, mqtt, tx_queue).

@joanwa Do you like to check the [wiki](https://emsesp.github.io/docs/) section MQTT and API and document the name changes?

**Comment by proddy** [comment]
> @proddy Due to this name changes i get a lot of mismatches when changing from esp8266 to esp32-v3. I would like to have it mostly compatible. Should i port this to v3? (i know it's manual merging). I'll also push the latest changes (WPA2, mqtt, tx_queue).

It would be great if you could manually merge into v3. I think it'll be quite complicated for @joanwa since the logic has changed. You've spent a lot of time on EMS-ESP these last days which is much appreciated. If you need any help let me know. I'll continue with the WebUI changes (#690) and after that work on the DE translation

**Comment by joanwa** [comment]
yeah, I was expecting some issues and discussions. Will do the documentation, and also manual merge into v3. Will also try to figure out the actual sensor values for MM100, and not re-broadcasted values for the low loss header.

File: src/devices/mixer.cpp (hunk 2 above)
Comment (joanwa) [reply src/devices/mixer.cpp]: ok, thanks for the info. In my setup, it's exactly the temperature from the low loss header. Will than have to find the actual sensor value in the MM100 telegrams. Already looking forward going through Norberts docu again :D

**Pull request merged**